package.json

```diff
@@ -69,7 +69,7 @@
     "tslint": "^5.10.0",
     "tslint-config-prettier": "^1.13.0",
     "typedoc": "^0.15.0",
-    "typescript": "^3.5.3"
+    "typescript": "3.5.3"
```

Comment:
Does this change have anything to do with the problem this PR resolves?

Reply:
Oh, I see your comment on this above. So this was just necessary to get the project to build correctly?

Reply:
this change isn't related, but as I mentioned above I wasn't able to get the project to build without it...